backend/docs/cron/run_visit_scheduling.py

```
@@ -64,64 +64,104 @@
+    try:
+        response = requests.post(search_url, headers=headers, json=search_request, timeout=60)
+        if response.status_code == 200:
+            data = response.json()
+            page_visits = data.get("ScheduledVisits", [])
+            total_count = data.get("TotalCount", 0)
+            if total_count > 0 and len(page_visits) == 0 and offset == 0:
+                print(f"Warning: TotalCount={total_count} but no visits found in response. Response keys: {list(data.keys())}")
+        else:
+            print(f"Search returned status {response.status_code}: {response.text[:200]}")
+            page_visits = []
+    except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)